index.js

```diff
@@ -92,14 +143,18 @@ const main = async () => {
       }
       if (
         key.Status === 'Active' &&
-        daysOld > daysError
+        daysOld >= daysError
       ) {
         console.log(`Found Expired Key: ${UserName} - ${key.AccessKeyId} - ${daysOld} days old`)
         expiredKeys.push({
           user: UserName,
           keyId: key.AccessKeyId,
           daysOld,
         })
+        if (daysOld == daysError || (daysOld-daysError)%12== 0) {
```

Comment:
In js world it's always preferable to use `===` instead of `==` for equality as `==` can be true in certain cases triple-equals isn't. You may already know this, but if, not just sharing a js tip as I know you're stronger in Python 😉 

Could be cool to extract this as a constant like `REMINDER_EMAIL_PERIOD` or `REMDER_EMAIL_DELAY`. It could be a top level `const`

Reply:
Re-reading this logic, both sides of the conditional will be truthy if `daysOld === daysError`. Maybe this could be simplified like...
```js
const daysPastExpiration = daysOld - daysError
if (daysPastExpiration % REMINDER_EMAIL_PERIOD === 0) {
```